conjur-api/Variable.cs

```diff
@@ -38,5 +39,25 @@ public string GetValue()
         {
             return this.Client.AuthenticatedRequest(this.path + "/value").Read();
         }
+
+        /// <summary>
+        /// Adds a value to the variable.
+        /// </summary>
```

Comment:
I know the documentation convention we've adopted here is verbose bordering on the ridiculous, but please for the sake of consistency add the `<param>` here.

Reply:
Done